src/generic/grideditors.cpp

```diff
@@ -1710,5 +1711,107 @@ wxGridCellAutoWrapStringEditor::Create(wxWindow* parent,
                                     wxTE_MULTILINE | wxTE_RICH);
 }
 
+#if wxUSE_DATEPICKCTRL
+
+// ----------------------------------------------------------------------------
+// wxGridCellDateEditor
+// ----------------------------------------------------------------------------
+
+wxGridCellDateEditor::wxGridCellDateEditor()
+{
+}
+
+void wxGridCellDateEditor::Create(wxWindow* parent, wxWindowID id,
+                                  wxEvtHandler* evtHandler)
+{
+    m_control = new wxDatePickerCtrl(parent, id);
+
+    wxGridCellEditor::Create(parent, id, evtHandler);
+}
+
+void wxGridCellDateEditor::SetSize(const wxRect& r)
+{
+    wxRect rect = r;
+    wxSize bestSize = m_control->GetBestSize();
```

Comment:
Minor, but it's preferable to make the local variables that don't change constant. I.e. `const wxSize bestSize = ...`.